core/ecschema-editing/src/Editing/Constants.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Editing
+ */
+
+import { ConstantProps, DelayedPromiseWithProps, Phenomenon, SchemaItemKey, SchemaItemType, SchemaKey, SchemaMatchType } from "@bentley/ecschema-metadata";
+import { SchemaContextEditor, SchemaItemEditResults } from "./Editor";
+import { MutableConstant } from "./Mutable/MutableConstant";
+import { MutableSchema } from "./Mutable/MutableSchema";
+
+/**
+ * @alpha
+ * A class allowing you to create schema items of type Constant.
+ */
+export class Constants {
+  public constructor(protected _schemaEditor: SchemaContextEditor) { }
+
+  public async create(schemaKey: SchemaKey, name: string, phenomenon: SchemaItemKey, definition: string, displayLabel?: string, numerator?: number, denominator?: number): Promise<SchemaItemEditResults> {
```

Comment:
These `XXXEditResults` types are awkward. They can consist of an error message, or a result, or neither, or both.
Are you intentionally avoiding using exceptions?
If so, can you define the result type so it can consist strictly of a result OR an error?